### PR TITLE
[documentation] Favor QT_QPA_PLATFORM over -platform. Fixes JB#48639

### PR DESCRIPTION
**Pull request by martyone**
Qt IFW is verbose about the -platform option.

### CODE REVIEW

File: doc/src/sdk-user-manual-noninteractive.qdoc [hunk 1]
```diff
@@ -67,8 +67,8 @@ maintenance.
         \endlist
 \endtable
 
-Execution on a headless system is possible with the help of the standard Qt
-option \c {-platform minimal}, likely in combination with \c --verbose.
+Execution on a headless system is possible after setting \c QT_QPA_PLATFORM=minimal in the
+environment, likely in combination with the \c --verbose option to the installer.
```
Comment (vigejolla) [inline doc/src/sdk-user-manual-noninteractive.qdoc]: I'd prefer using the term "environment variable"

**Review by vigejolla — CHANGES_REQUESTED** [review]

File: doc/src/sdk-user-manual-noninteractive.qdoc (hunk 1 above)
Comment (martyone) [reply doc/src/sdk-user-manual-noninteractive.qdoc]: There is not only the variable name but also the specific value mentioned, so I preferted this formulation over "after setting the environment variable \c QT_QPA_PLATFORM to \c minimal" or similar. One cannot simply copy & paste that :)

Comment (vigejolla) [reply doc/src/sdk-user-manual-noninteractive.qdoc]: Just add the variable to the examples section below and the problem is solved. So in this section, say
`Execution on a headless system is possible after setting the \c QT_QPA_PLATFORM environment variable to value "minimal", likely in combination with the \c --verbose option to the installer. ` and then in the examples section
`$ QT_QPA_PLATFORM=minimal ./installer.run --verbose non-interactive=1 accept-licenses=1`

Comment (martyone) [reply doc/src/sdk-user-manual-noninteractive.qdoc]: That would hurt readability. Updated.

**Review by vigejolla — APPROVED** [review]
LGTM

**Pull request merged**